pcs_test/tier0/test_lint.py

```diff
@@ -0,0 +1,40 @@
+import os
+import re
+from unittest import TestCase
+
+PACKAGE_DIR = os.path.realpath(
+    os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+)
```

Comment:
This will not work with `--installed` option. IMHO the easiest way to solve this would be to use `import pcs; pcs.__path__`